integrations/lambda/README.md

```diff
@@ -0,0 +1,108 @@
+# Timestream for LiveAnalytics Lambda Sample Application
+
+## Overview
+
+This sample application demonstrates how [time series data](https://docs.aws.amazon.com/timestream/latest/developerguide/concepts.html) can be ingested into [Timestream for LiveAnalytics](https://docs.aws.amazon.com/timestream/latest/developerguide/what-is-timestream.html) using an [AWS Lambda function](https://aws.amazon.com/lambda/) and [`boto3`](https://boto3.amazonaws.com/v1/documentation/api/latest/index.html).
+
+This sample application is comprised of three files:
+- `demo.ipynb`: A [Jupyter notebook](https://jupyter.org/) that:
+    - Generates simulated time series data from a selection of predefined scenarios or a user-defined scenario.
+    - Deploys a Lambda function that receives the data and ingests the data into Timestream for LiveAnalytics.
+    - Sends the generated time series data to the Lambda's URL using SigV4 authentication.
+- `dashboard.json`: A Grafana dashboard, configured to view all data ingested into the Timestream for LiveAnalytics database in the last hour.
+- `requirements.txt`: A file containing required packages for the Jupyter notebook, for quick environment setup.
+
+The following diagram depicts the deployed Lambda function receiving generated data and ingesting the data to Timestream for LiveAnalytics that then is queried and displayed in [Amazon Managed Grafana](https://aws.amazon.com/grafana/).
+
+<img src="./img/lambda_ingestion_overview.png" width=575px/>
+
+## Prerequisites
+
+1. [Configure AWS credentials for use with boto3](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html).
+2. [Install Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html).
+3. Initialize a Conda environment named `sample_app_env` with the required packages by running:
+    ```shell
+    conda env create -f environment.yml
+    ```
+4. Activate the environment by running:
+    ```shell
+    conda activate sample_app_env
```

Comment:
I had the following issues when executing this.

`CondaError: Run 'conda init' before 'conda activate'`

To fix I did the following:
`conda init zsh`
`source ~/.zshrc`
`conda activate sample_app_env`

Commands will differ depending if using zsh/bash

Reply:
Added.